xep-0369.xml

```diff
@@ -637,9 +637,6 @@
     to='mix.shakespeare.example'
     from='hag66@shakespeare.example/pda'
     id='7nve413p'>
-  <register xmlns='urn:xmpp:mix:0'>
```

Comment:
I will change the commit message, but the change should indeed stay.

Also github still randomly fails to send email notifications… /rant